.github/workflows/app_build.yml

```diff
@@ -8,19 +8,20 @@ on:
       - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
   pull_request:
     branches: [ master, develop ]
+    types: [ opened, reopened, synchronize, ready_for_review ]
```

Comment:
Why do you want to add this?

Reply:
They default types for pull_requests are [ opened, reopened, synchronize ].  I added ready_for_review in the case that a PR is draft and then you mark it ready for review without pushing a commit.  The idea is if you want to push code to a branch but not kick off a bunch of CI builds you can mark the PR as draft.  Then when you want CI to run you set as ready for review.  To me that seems nicer than adding [skip ci] to each commit message but we could do that too.